client/src/components/ui/tag/DisplayTags.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Tag } from 'antd';
+import { TAG_PAGE } from 'constants/links';
+
+const { CheckableTag } = Tag;
+
+class DisplayTags extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tags: this.props.tags,
+    }
+  }
+
+  handleChange(tag, checked) {
+    const { selectedTags } = this.state;
+    const nextSelectedTags = checked ? [...selectedTags, tag] : selectedTags.filter(t => t !== tag);
+    this.setState({ selectedTags: nextSelectedTags });
+  }
+
+  componentDidUpdate = () => {
+    if (this.state.tags != this.props.tags) 
```

Comment:
How about '!==' instead of '!='? It will not make an error but just for stability.